addon/components/au-dropdown.hbs

```diff
@@ -18,18 +19,30 @@
     </span>
   </AuButton>
   <div
-    class="au-c-dropdown__menu {{this.alignment}} {{if this.dropdownOpen 'is-visible'}}"
-    role="menu"
-    tabindex="-1"
-    {{focus-trap
-      isActive=this.dropdownOpen
-      shouldSelfFocus=true
-      focusTrapOptions=(hash
-        clickOutsideDeactivates=this.clickOutsideDeactivates
-      )
+    {{
+      au-floating-ui
```

Comment:
I would import the modifier in the backing class and store it on a property. That aligns better with the future strict mode components and means the modifier is truly for internal use only for now.

Reply:
just to be sure, is the following code example in line with what you meant?

`au-dropdown.js`
```js
import AuFloatingUiModifier from '@appuniversum/ember-appuniversum/modifiers/au-floating-ui';

export default class AuDropdown extends Component {
  auFloatingUi = AuFloatingUiModifier;

  // etcetera
}
```

`au-dropdown.hbs`
```handlebars
<div
  {{this.auFloatingUi
    this.referenceElement
    this.arrowElement
    defaultPlacement=this.alignment
    options=this.floatingUiOptions
  }}
>
  <!-- etcetera -->
</div>
```

Reply:
Yup! With some small nitpicks. I would only move the file to a "private" folder or something (and drop the `Au` prefix since it's only used internally anyways). So something like this:

```js
import FloatingUiModifier from '@appuniversum/ember-appuniversum/private/modifiers/floating-ui';
```

Reply:
Oh, I think you mentioned this a while ago. I'll get it done 👍 Thanks!